### DISCUSSION TITLE
Surface Desert Mob Spawners

### DISCUSSION THREAD

**Discussion by jbredwards** [ACGaming/UniversalTweaks#98] [Tweak Suggestions]
This is very niche, but in older versions of the game [these](https://bugs.mojang.com/secure/attachment/53352/dungeon%202%20before.png.jpg) would occasionally generate. It would be nice to have them back :)

**Comment by ACGaming** [comment]
Did some research and people reported that these still exist but are not as apparent anymore because world generation doesn‘t send updates for falling blocks nowadays.
